games/minimal/game.conf

```diff
@@ -1,2 +1,2 @@
 name = Minimal development test
-
+description = Testing environment to help with testing the engine features of Minetest. It can also be helpful in mod development.
```

Comment:
Can probably be shortened a bit to something along the lines of `Minimal testbed for engine or mod development`